pkgs/desktops/mate/libmatekbd/default.nix

```diff
@@ -1,35 +1,47 @@
-{ lib
-, stdenv
-, fetchurl
-, pkg-config
-, gettext
-, gtk3-x11
-, libxklavier
-, mateUpdateScript
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  glib,
+  gtk3,
+  gobject-introspection,
+  libxklavier,
+  nix-update-script,
 }:
 
 stdenv.mkDerivation rec {
   pname = "libmatekbd";
   version = "1.28.0";
 
-  src = fetchurl {
-    url = "https://pub.mate-desktop.org/releases/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    sha256 = "XS5YSDwrI9M1A9JMiPi5CijMAYnX5AAbPic6YE9v6A4=";
+  src = fetchFromGitHub {
+    owner = "mate-desktop";
+    repo = "libmatekbd";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-6s8JiuXbBWOHxbNSuO8rglzOCRKlQ9fx/GsYYc08GmI=";
   };
 
+  strictDeps = true;
+
+  depsBuildBuild = [ pkg-config ];
+
   nativeBuildInputs = [
+    meson
+    ninja
     pkg-config
-    gettext
+    glib
+    gobject-introspection
   ];
 
   buildInputs = [
-    gtk3-x11
```

Comment:
I think `gdk/gdkx.h` requires gtk3-x11 on non-linux, we can see if ofborg builds darwin, ~~maybe we should just mark mate packages linux only~~ :shrug: